server/lib/chat/chat-service.test.ts

```diff
@@ -921,14 +914,7 @@ describe('chat/chat-service', () => {
         })
         expect(client2.publish).toHaveBeenCalledWith(getChannelPath(testChannel.id), {
           action: 'message2',
-          message: {
-            id: textMessage.msgId,
-            type: textMessage.data.type,
-            channelId: textMessage.channelId,
-            from: textMessage.userId,
-            time: Number(textMessage.sent),
-            text: processedMessageString,
-          },
+          message: toTextMessageJson(textMessage),
           user: {
```

Comment:
Dunno how many times you do this throughout but is it really necessary to split these fields out? can't you just do `user: user1`? It'd be kind of vastly preferable to do that because if/when we add more fields to `SbUser` such tests will all have to be updated, not something that we really want to have to do